### PR TITLE
New design WIP

### PR DESCRIPTION
**Pull request by Pierre-LHOSTE**
| Light | Dark |
| - | - |
| Capture d’écran 2023-12-20 à 16 29 50 | Capture d’écran 2023-12-20 à 16 28 32 |

### CODE REVIEW

File: src/components/RestaurantBadge.jsx
Comment (alexsegura) [inline src/components/RestaurantBadge.jsx:1]: No need to use the `.jsx` extension, use `.js`

File: src/components/RestaurantCard.jsx
Comment (alexsegura) [inline src/components/RestaurantCard.jsx:1]: No need to use the `.jsx` extension, use `.js`

File: src/components/RestaurantList.js [hunk 1]
```diff
@@ -9,7 +9,9 @@ import { getRestaurantCaption, shouldShowPreOrder } from '../utils/checkout'
 import {
   CategoryBadge,
   TimingBadge,
-} from '../navigation/checkout/components/RestaurantBadges'
+} from '../navigation/checkout/components/RestaurantBadges';
+import {RestaurantCard} from './RestaurantCard';
+import {RestaurantBadge} from './RestaurantBadge';
```
Comment (alexsegura) [inline src/components/RestaurantList.js:14]: Doesn't seem to be used in this file

**Comment by vladimir-8** [comment]
Great work! @Pierre-LHOSTE please add Android screenshots too 😉

**Comment by alexsegura** [comment]
@vladimir-8 

| Light | Dark |
| - | - |
| Screenshot_1703240913 | Screenshot_1703241123 |

File: src/i18n/locales/fr.json [hunk 2]
```diff
@@ -465,6 +465,9 @@
         "CHECKOUT_LOOPEAT_ADD_CREDITS": "Créditer ma cagnotte",
         "RESTAURANT_LOOPEAT_DISCLAIMER": "Changez le nombre d'emballages utilisés pour emballer la commande",
         "ZERO_WASTE": "Zéro déchet",
+        "FEATURED": "À la une",
+        "EXCLUSIVE": "Exclusivités",
+        "NEW": "Nouveautés",
```
Comment (alexsegura) [inline src/i18n/locales/fr.json:470]: Can you also add those translations to the English language?

File: src/i18n/locales/eu.json [hunk 3]
```diff
@@ -386,6 +386,10 @@
         "SELECT_PAYMENT_METHOD": "Hautatu ordainketa metodo bat",
         "PRIVACY": "Pribatutasuna",
         "TERMS_OF_SERVICE": "Zerbitzu-baldintzak",
-        "CHECKOUT_UNITS": "Unitateak"
+        "CHECKOUT_UNITS": "Unitateak",
+        "ZERO_WASTE": "Zero waste",
+        "FEATURED": "Featured",
+        "EXCLUSIVE": "Exclusive",
+        "NEW": "New"
```
Comment (alexsegura) [inline src/i18n/locales/eu.json:393]: This is the Euskadi (Basque) translation file, not English (`en.json`)

**Review by alexsegura — APPROVED** [review]

**Pull request merged**